lib/ansible/modules/cloud/amazon/ec2_eni.py

```diff
@@ -79,20 +79,24 @@
     version_added: 2.2
   force_detach:
     description:
-      - Force detachment of the interface. This applies either when explicitly detaching the interface by setting instance_id to None or when deleting an interface with state=absent.
+      - Force detachment of the interface. This applies either when explicitly detaching the interface by setting \
```

Comment:
If you align the text on the second line with the text of the first line, you don't need line break markers.

e.g.

```
  property:
    description:
      - Pretend these are
        actually really long 
        lines and you can 
        avoid line breaks
```